extensions/notebook/src/book/bookTrustManager.ts

```diff
@@ -0,0 +1,137 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
this class looks quite test-able. Maybe during committer time tests could be added 😄?

Reply:
I am a little stuck with the Debugging Extension Unit Tests hanging indefinitely. Need to follow up with team members on Thursday to see if I can get unblocked.

Reply:
You can also do print (console) debugging too - that's what I usually fall back on when I start having issues debugging (I have this kind of thing happen occasionally too for the extension tests, haven't had time to look into it much yet unfortunately)

Reply:
Gotcha. I managed to run the tests by getting rid of the rest of the modules in the `scripts\test-extensions-unit.bat` script.